File: leet_991_broken_calculator__Recur.py
class Solution:
    def brokenCalc(self, start: int, target: int) -> int:
        if start >= target:
            return start - target
        if target % 2 == 0:
            return 1 + self.brokenCalc(start, target // 2)
        return 1 + self.brokenCalc(start, target + 1)

File: leet_991_broken_calculator__Recur.js
/**
 * @param {number} startValue
 * @param {number} target
 * @return {number}
 */
var brokenCalc = function(start, target) {
    if (start >= target)
        return start - target
    if (target % 2 == 0)
        return 1 + brokenCalc(start, Math.floor(target / 2))
    return 1 + brokenCalc(start, target + 1)
};
